app/views/api/v1/models/_inbox.json.jbuilder

```diff
@@ -22,6 +22,9 @@ json.phone_number resource.channel.try(:phone_number)
 json.selected_feature_flags resource.channel.try(:selected_feature_flags)
 json.reply_time resource.channel.try(:reply_time)
 json.reauthorization_required resource.channel.try(:reauthorization_required?) if resource.facebook?
+json.agent_bot do
```

Comment:
I would advise against this. If we need to fetch whether there is an agent bot associated with the inbox, we can fetch it separately. 

Adding it here would create unnecessary DB queries in the dashboard while fetching inboxes or a particular inbox.

Reply:
Also, this might be an N+1 query.

Reply:
Ideally inbox/{inbox_id}/agent_bots is a good way to do it.